pages/treeherder.py

```diff
@@ -51,6 +59,42 @@ def open_perfherder_page(self):
         from perfherder import PerfherderPage
         return PerfherderPage(self.base_url, self.selenium).wait_for_page_to_load()
 
+    def select_next_job(self):
+        el = self.selenium.find_element(*self._resultset_locator)
+        Wait(self.selenium, self.timeout).until(EC.visibility_of(el))
+        el.send_keys(Keys.ARROW_RIGHT)
+        Wait(self.selenium, self.timeout).until(lambda s: self.job_result_status)
```

Comment:
This works when there isn't already a job selected, but if you called `select_next_job` twice, the second time this wait condition would evaluate as true potentially before the new job is selected.